src/main/resources/csv/mainnet.csv

```diff
@@ -1,3 +1,2 @@
 id|name|standard|account_id
-1 | USDTez | FA1.2 | tz1eEnQhbwf6trb8Q8mPb2RaPkNk2rN7BKi9
-
+1 | USDTez | FA1.2 | KT1RmDuQ6LaTFfLrVtKNcBJkMgvnopEATJux
```

Comment:
Please rename this file to `src/main/resourcs/registered_tokens/babylonnet.csv` and create corresponding blank files for mainnet and carthaganet. Please make sure the `registered_tokens` table logic is modified correctly to use these files.

Reply:
Done, also, I extracted the csv parsing out of the main Lorre code into the TokenContracts class.

The file will be chosen based on the current network name used in the configuration for Lorre